nft-client/src/main.rs

```diff
@@ -460,8 +536,8 @@ async fn main() -> anyhow::Result<()> {
                 .await
                 .context("Failed to connect to Node GRPC")?;
 
-            // Measured by experiments, will likely break if contract state becomes large:
-            let estimated_energy = 300 * tokens_len + 2000;
+            // Measured to be sufficient by experiments, will depend on the contract:
+            let estimated_energy = 3000;
```

Comment:
Wouldn't it make more sense for the user to be able to supply it? (and default to 10000 or some such).